src/emittermixin.js

```diff
@@ -150,14 +156,18 @@ const EmitterMixin = {
 		//     ...
 		// }
 
-		if ( !( emitters = this._listeningTo ) ) {
-			emitters = this._listeningTo = {};
+		if ( !this[ _listeningTo ] ) {
+			this[ _listeningTo ] = {};
 		}
 
-		if ( !( emitterId = emitter._emitterId ) ) {
-			emitterId = emitter._emitterId = uid();
+		emitters = this[ _listeningTo ];
+
+		if ( !emitter._emitterId ) {
+			emitter._emitterId = uid();
```

Comment:
This got complicated in the process.

`_emitterId` is available through setter and getter (which are protected). If `_emitterId` is only a private property I should not write `emitter[ _emitterId ]` because then, from "this" emitter I am reaching `_emitterId` of `emitter` (different object).

That's why I introduced getter and setter which are protected. But to "hide" the property itself (so it's not enumerable) I still kept `_emitterId` `Symbol`.